### DISCUSSION TITLE
Color scheme for the application

### DISCUSSION THREAD

**Discussion by Abhishek-Sathe-Tafe** [ZiyangWang001007081/Tafe230243#3] [General]
Hi there,

Are you able to share some screenshots for your current conversion calculator? I want to match up the color scheme on my calculator

**Comment by ZiyangWang001007081** [comment]
Hi Abhishek,

I have attached my CurrentConversion Calculator for reference. I copied the \<PageResources\> tags from MainMenu Page to my CurrencyConverter Page to keep the style consistent.
![image](https://github.com/ZiyangWang001007081/Tafe230243/assets/151332477/1a9dc2f9-9f27-4664-a14a-838bc30c511c)

Furthermore, I have added the following Universal style for TextBoxes and drop-down boxes:
![image](https://github.com/ZiyangWang001007081/Tafe230243/assets/151332477/4d44c7c8-b282-4d2e-87c3-c7eaa99c711c)

Feel free to implement these styles in your page if needed.

**Comment by Abhishek-Sathe-Tafe** [comment]
Well done. Great work! I'll try to use the same style. Thank you :)
